CourseManager/Courses/Course 4 UI and Animations in SwiftUI/Views/Components/ForecastCard.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ForcastCard.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-10.
+//
+
+import SwiftUI
+
+struct ForecastCard: View {
+    var forecast: Forecast
+    var forecastPeriod: ForecastPeriod
+    var isActive: Bool {
+        if forecastPeriod == ForecastPeriod.hourly {
+            let isThisHour = Calendar.current.isDate(.now, equalTo: forecast.date, toGranularity: .hour)
```

Comment:
Here isThisHour is extra, because the method already tells the whole idea

Reply:
agreed, updated